fischkapp/src/components/Header.tsx

```diff
@@ -1,16 +1,24 @@
 import { StyledHeader, StyledLogo } from "./styles/Header.styled";
 import addButton from "../assets/addButton.svg";
 import { StyledAddButton } from "./styles/buttons/AddButton";
-import { flashCardsData } from "../data/data";
-export const Header = () => {
+
+type HeaderPropTypes = {
+  onSetNewCard: Function;
```

Comment:
**nitpick:** This function is typed incorrectly, as it does not say what arguments this function accepts. It should be:

```
type HeaderPropTypes = {
  onSetNewCard: (isSettingCard: boolean) => void;
```